themes/classic/_dev/js/components/product-miniature.js

```diff
@@ -25,28 +25,35 @@
 import $ from 'jquery';
 
 export default class ProductMinitature {
-  init(){
+  init() {
     $('.js-product-miniature').each((index, element) => {
       const FLAG_MARGIN = 10;
-      let $discount = $(element).find('.discount-product');
-      let $onsale =  $(element).find('.on-sale');
-      let $new = $(element).find('.new');
-      if($discount.length){
-        $new.css('top', $discount.height() * 2 + FLAG_MARGIN);
-        $discount.css('top',-$(element).find('.thumbnail-container').height() + $(element).find('.product-description').height() + FLAG_MARGIN);
+      const discountElems = $(element).find('.discount-product');
+      const onSaleElems =  $(element).find('.on-sale');
+      const newElems = $(element).find('.new');
+
+      if (discountElems.length) {
+        newElems.css('top', discountElems.height() * 2 + FLAG_MARGIN);
+        discountElems.css('top', -$(element).find('.thumbnail-container').height() + $(element).find('.product-description').height() + FLAG_MARGIN);
+
+        if ($(element).find('.pack').length) $(element).find('.pack').css('top', discountElems.height() * 2 + FLAG_MARGIN);
```

Comment:
Can you please use braces here? Even if there is only one line, it will increase the readbility.

```js
if (...) {
   [...]
}
```

Reply:
And it's eslint compliance

Reply:
@Quetzacoalt91 added in https://github.com/PrestaShop/PrestaShop/pull/10267/commits/4b2f8bc7906d0103c4b37fd7bbcf0169571f931c
also rebased for assets.